docs/modules/ROOT/pages/documentation/payara-server/upgrade-server.adoc

```diff
@@ -0,0 +1,59 @@
+[[Upgrade Server]]
+= Upgrade Payara Server
+
+Payara Server can be upgraded to new versions from version *5.24.0*
+
+NOTE: This feature is in technical preview.
+
+[[upgrade-server-command]]
+== Upgrade Server Command
+
+*Usage*::
+`asadmin> upgrade-server`
+
+*Aim*::
+This command can be used to upgrade Payara Server to the specified distribution along with all SSH nodes in the _domain.xml_.
+The command will also back up the old version and domains in the default location (_payara5/glassfish/domains_) to allow for rolling back. The command can also be used to downgrade to an older Payara version although this is not encouraged.
+
+[[command-options-1]]
+=== Command Options
+
+[cols=",,,,",options="header",]
+|=======================================================================
+|Option |Type |Description |Default |Mandatory
+|`--username` |String | The username for the Payara Nexus server.||Yes
+|`--version` |String |Specifies the version number of the new version of Payara to use.| |Yes
+|`--distribution` |Enum |Specifies the distribution of Payara Server to download. This can payara, payara-web, payara-ml, or payara-web-ml.|payara|No
+|=======================================================================
+
+[[example-5]]
+=== Example
+
+This example upgrades a Payara Web distribution to version 5.24.1
+
+[source, shell]
+----
+asadmin> upgrade-server --username example-user --distribution web --verson 5.24.1
+----
+
+
+NOTE: Upgrading Payara Server is not currently supported on Windows.
+
+[[Rollback-server-command]]
+== Rollback Server Command
+
+*Usage*::
+`asadmin> rollback-server`
+
+*Aim*::
+This command can be used to rollback Payara Server to the point before the upgrade-server command was run. This does not rollback the _domain.xml_.
+
+[[example-5]]
```

Comment:
Bit of a pointless example as it is. Either drop it or add in an extra couple of lines of unzipping a backed up domain.xml and moving it back in to the correct place.